lib/full-schema.js

```diff
@@ -148,6 +148,6 @@ export const marks = {
 const schemaWithoutListNodes = new Schema({nodes, marks});
 
 export const schema = new Schema({
-  nodes: addListNodes(schemaWithoutListNodes.spec.nodes, 'paragraph block*', 'block'),
+  nodes: addListNodes(schemaWithoutListNodes.spec.nodes, 'paragraph+ list_element*', 'block list_element'),
```

Comment:
So calling `addListNodes` does not add `list_element` automatically?

Reply:
`list_element` was just a group that I created for this schema (it wasn't defined in `prosemirror-schema-list`). Now I changed the `itemContent` argument value to `(ordered_list | bullet_list) ` instead of `list_element`.